wagtailflags/templates/wagtailflags/flags/edit_condition.html

```diff
@@ -20,34 +17,34 @@ <h2 class="nice-padding">
             {% for field in form %}
             <li {% if field.field.required %}class="required"{% endif %}>
                     {% if not field.is_hidden %}
-                        <div class="field">
-                            {{ field.label_tag }}
 
-                            <div class="field-content">
-                                <div class="input">
-                                    {{ field }}
-                                </div>
-                                {% if field.help_text %}
-                                    <p class="help">{{ field.help_text|safe }}</p>
-                                {% endif %}
-                            </div>
+                    <div class="w-field__wrapper " data-field-wrapper="">
```

Comment:
This introduces a lot of duplication of Wagtail's [panel.html](https://github.com/wagtail/wagtail/blob/v4.0/wagtail/admin/templates/wagtailadmin/shared/panel.html) introduced in 4.0. I tried this and it seems to work reasonably:

```html
{% include "wagtailadmin/shared/field.html" with field=form.condition %}
{% include "wagtailadmin/shared/field.html" with field=form.value %}
{% include "wagtailadmin/shared/field.html" with field=form.required %}
```

(I don't think we can use a `{% for %}` loop as above in create_flag.html because we want to exclude the `flag` field since it gets set by the POST URL.)

Reply:
Done in [684ee68](https://github.com/cfpb/wagtail-flags/pull/72/commits/684ee6857d85a03041d2e8b0a7fc8a695d40c852).